provider/pkg/provider/provider.go

```diff
@@ -2672,9 +2684,11 @@ func pruneLiveState(live, oldInputs *unstructured.Unstructured) *unstructured.Un
 	return oldLivePruned
 }
 
-// shouldNormalize returns false for CRDs and CustomResources, and true otherwise.
+// shouldNormalize returns false for CRDs, CustomResources, and List kinds and true otherwise.
 func shouldNormalize(uns *unstructured.Unstructured) bool {
-	return !clients.IsCRD(uns) && kinds.KnownGroupVersions.Has(uns.GetAPIVersion())
+	return !clients.IsCRD(uns) &&
+		kinds.KnownGroupVersions.Has(uns.GetAPIVersion()) &&
+		!strings.HasSuffix(uns.GetKind(), "List")
```

Comment:
Why are we checking the suffix here but using the `ListQualifiedTypes` lookup above in `isListURN`?

Reply:
This is a redundant check with the added logic in the Check method, and it was nontrivial to plumb the URN through, or calculate it from the Unstructured object. The first two conditions already cover the case of CustomResources that happen to have a List suffix, so this is an easy way to add a backstop.

Reply:
Thinking about it some more, I'm actually going to remove this check entirely.